src/ka/validation.php

```diff
@@ -13,41 +14,41 @@
     */
 
     'accepted'             => ':attribute უნდა იყოს მონიშნული.',
-    'active_url'           => ':attribute უნდა იყოს URL მისამართი.',
-    'after'                => ':attribute უნდა იყოს :date-ის შემდეგ.',
-    'after_or_equal'       => ':attribute უნდა იყოს :date-ის შემდეგ ან მისი ტოლი.',
+    'active_url'           => ':attribute არ არის სწორი URL მისამართი.',
+    'after'                => ':attribute უნდა იყოს თარიღი :date-ის შემდეგ.',
+    'after_or_equal'       => ':attribute უნდა იყოს თარიღი :date-ის შემდეგ ან მისი ტოლი.',
     'alpha'                => ':attribute უნდა შეიცავდეს მხოლოდ ასოებს.',
-    'alpha_dash'           => ':attribute უნდა შეიცავდეს მხოლოდ ასოებს, რიცხვებს და ტირეებს.',
+    'alpha_dash'           => ':attribute უნდა შეიცავდეს მხოლოდ ასოებს, რიცხვებს, ტირეებს და ქვეტირეებს.',
     'alpha_num'            => ':attribute უნდა შეიცავდეს მხოლოდ ასოებს და რიცხვებს.',
     'array'                => ':attribute უნდა იყოს მასივი.',
-    'before'               => ':attribute უნდა იყოს :date-მდე.',
-    'before_or_equal'      => ':attribute უნდა იყოს :date-მდე ან მისი ტოლი.',
+    'before'               => ':attribute უნდა იყოს თარიღი :date-მდე.',
+    'before_or_equal'      => ':attribute უნდა იყოს თარიღი :date-მდე ან მისი ტოლი.',
     'between'              => [
         'numeric' => ':attribute უნდა იყოს :min-სა და :max-ს შორის.',
         'file'    => ':attribute უნდა იყოს :min-სა და :max კილობაიტს შორის.',
         'string'  => ':attribute უნდა იყოს :min-სა და :max სიმბოლოს შორის.',
         'array'   => ':attribute-ის რაოდენობა უნდა იყოს :min-დან :max-მდე.',
     ],
-    'boolean'              => ':attribute უნდა იყოს true, false, 0 ან 1.',
-    'confirmed'            => ':attribute არ ემთხვევა დადასტურებას.',
+    'boolean'              => ':attribute ველი უნდა იყოს true ან false.',
+    'confirmed'            => ':attribute-ის დადასტურება არ ემთხვევა.',
```

Comment:
:attribute-ის it's working ?

Reply:
yes, of course.

![regex](https://user-images.githubusercontent.com/56151710/104440380-b6d34600-55ab-11eb-9118-fe0a819de6aa.png)

Reply:
I tested it on regex and :attribute-ის is correct  and it's working properly.